Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertAdaptiveComponent.cs

```diff
@@ -66,7 +66,7 @@ public ApplicationObject AdaptiveComponentToNative(AdaptiveComponent speckleAc)
 
         isUpdate = true;
       }
-      catch
+      catch (Autodesk.Revit.Exceptions.ApplicationException)
       {
         //something went wrong, re-create it
       }
```

Comment:
If we fail here, should we notify the user (i.e update the appObj log) or just fail.

If this fails, I'm assuming the user would end. up with 2 adaptive components in the document instead of 1?